pkg/dashboard/server/agentApi.go

```diff
@@ -212,6 +219,22 @@ func update(w http.ResponseWriter, r *http.Request) {
 	}
 	w.WriteHeader(http.StatusOK)
 
+	if update.Event == agent.EventPodCreated || update.Event == agent.EventPodUpdated {
```

Comment:
What to do with pod deleted updates?
I would think it needs to be deleted from the pod state manager too.. or?